index.js

```diff
@@ -8,43 +8,65 @@ var hasFlag = function (flag) {
 	return pos !== -1 && (terminator !== -1 ? pos < terminator : true);
 };
 
-module.exports = (function () {
-	if ('FORCE_COLOR' in process.env) {
-		return true;
+var support = function (level) {
+	if (level <= 0) {
+		return null;
```

Comment:
If we use `false` here, `supportsColor.has16` will work even though it's `false`.